tests/utils.py

```diff
@@ -32,6 +32,6 @@ def write_env_config(prefix, force=False, **kwargs):
     condarc = Path(prefix) / ".condarc"
     if condarc.is_file() and not force:
         raise RuntimeError(f"File {condarc} already exists. Use force=True to overwrite.")
-    yaml = YAML(typ="unsafe", pure=True)
+    yaml = YAML(typ="full", pure=True)
```

Comment:
See [`ruamel.yaml` deprecation](https://pypi.org/project/ruamel.yaml/):

> There will be at least one more potentially breaking change in the 0.18 series: `YAML(typ='unsafe')` now has a pending deprecation warning and is going to be deprecated, probably before the end of 2023. If you only use it to dump, please use the new `YAML(typ='full')`, the result of that can be safely loaded with a default instance `YAML()`, as that will get you inspectable, tagged, scalars, instead of executed Python functions/classes.